src/pybind/mgr/dashboard/frontend/src/main.ts

```diff
@@ -10,4 +11,12 @@ if (environment.production) {
 
 platformBrowserDynamic()
   .bootstrapModule(AppModule)
+  .then((moduleRef) => {
+    if (isDevMode()) {
+      const applicationRef = moduleRef.injector.get(ApplicationRef);
+      const componentRef = applicationRef.components[0];
+      // allows to run `ng.profiler.timeChangeDetection();`
```

Comment:
According to the comment here i assume you found this information on https://medium.com/@dmitrymogilko/profiling-angular-change-detection-c00605862b9f. If so, i think we should honor that and you should mention the website at least in the comment message.

Reply:
Done.